community/xkb-switch/build

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh -e
+
+export DESTDIR="$1"
+
+cmake -B build \
+    -DCMAKE_INSTALL_PREFIX=/usr
+
+cmake --build   build
+cmake --install build
+
+# man page may be compressed, install manually instead
+rm -rf "$1/usr/share/man"
```

Comment:
Won't the `install` command overwrite the man page anyway?

Reply:
man page is installed with .xz or .gz extension, does not get overridden because of this.

Reply:
You're right. :+1: